task/label-check/0.1/label-check.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
   name: label-check
 spec:
   description: >-
-    Checks that image carries best practice labels.
+    Utilizes Conftest to check if image has best practice labels.
```

Comment:
Suggestion: Verify whether an image contains the best practice labels with Conftest. For example...?

Question:

Do we need to add some context to Conftest?

Reply:
looks good, thanks.
I don't think it'd be a good idea to add examples here. Because we want to keep `spec.description`s strictly to the point and just convey what is happening in the task. For more info, users can always read the README.md files of each task that were added as part of these 2 PRs: https://github.com/redhat-appstudio/build-definitions/pull/330 and https://github.com/redhat-appstudio/build-definitions/pull/352

FWIW, if someone asks about the labels used for checks, this policy file would be a good (if not perfect) idea to share: https://github.com/redhat-appstudio/hacbs-test/blob/main/policies/image/deprecated-labels.rego#L4